src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -400,6 +402,7 @@ public void AddService<TService>(TService serviceInstance)
         /// Add an extension to register new binding attributes and converters.
         /// </summary>
         /// <param name="extension"></param>
+        [Obsolete("Not ready for public consumption.")]
         public void AddExtension(IExtensionConfigProvider extension)
```

Comment:
Probably keep this one. It's stable. Script will need it

Reply:
ok